espnet2/tasks/abs_task.py

```diff
@@ -1064,7 +1064,6 @@ def print_config(cls, file=sys.stdout) -> None:
     @classmethod
     def main(cls, args: argparse.Namespace = None, cmd: Sequence[str] = None):
         assert check_argument_types()
-        print(get_commandline_args(), file=sys.stderr)
```

Comment:
Why removing this?

Reply:
@pyf98 
I'm sorry, I accidentally deleted this line...
I just reverted this modification.